### DISCUSSION TITLE
How to use the API

### DISCUSSION THREAD

**Discussion by henrikrox** [shibayan/appservice-acmebot#295] [Q&A] [answered]
Hi so i tried using the API with

```
POST
https://func-xxxxxxx.azurewebsites.net
PATH
api/certificate

Headers
Content-Type - applicaion/json
x-functions-key - xxxxxxxxxxxx

Body
{
  "AppName": "xxxx",
  "ResourceGroupName": "xxxx",
  "SlotName": "xxx",
  "DnsNames": [
    "xxxxx"
  ]
}
```

I get a HTTP 202, but how can i use this to add the certificate to the appname? Maybe a stupid question perhaps

**Comment by shibayan** [comment, marked as answer]
I'll write up how to use it in the Wiki, so wait for me.

**Reply by shibayan** [reply]
I have written a minimal reference.
https://github.com/shibayan/appservice-acmebot/wiki/REST-API

**Comment by haespino** [comment]
Hola, ya cuento con un dashbord y la renovacion de certificados. Quisiera acceder a ella desde la api para leer la informacion de los dns en el. Como puedo authenticarme con Python. Muchas gracias. Saludos.
